blacklight-cornell/app/assets/stylesheets/singlesearch.scss

```diff
@@ -162,7 +162,8 @@ div.view-all {
     font-size: 1rem;
   }
   i {
-    color: $concrete;
+    // color: $concrete;
```

Comment:
same comment as above, re: cleaning up commented out css.

Reply:
yes... should clean up my WIP!